modules/raft/pom.xml

```diff
@@ -38,6 +38,11 @@
             <artifactId>ignite-core</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
raft should know nothing neither replicas nor transactions.

Reply:
Fixed.